refaster-support/src/test/java/tech/picnic/errorprone/refaster/matchers/RequiresComputationTest.java

```diff
@@ -8,54 +8,67 @@
 import com.sun.source.tree.ReturnTree;
 import org.junit.jupiter.api.Test;
 
-final class IsLikelyTrivialComputationTest {
+final class RequiresComputationTest {
   @Test
   void matches() {
     CompilationTestHelper.newInstance(MatcherTestChecker.class, getClass())
         .addSourceLines(
             "A.java",
+            "import java.io.OutputStream;",
             "import java.util.function.Predicate;",
             "",
             "class A {",
-            "  String negative1() {",
-            "    return String.valueOf(1);",
+            "  int negative1() {",
+            "    int[] arr = new int[0];",
+            "    return arr[0];",
             "  }",
             "",
             "  String negative2() {",
-            "    return toString().toString();",
+            "    return null;",
             "  }",
             "",
-            "  String negative3() {",
-            "    return \"foo\" + toString();",
+            "  boolean negative3() {",
+            "    return false;",
             "  }",
             "",
-            "  byte negative4() {",
-            "    return \"foo\".getBytes()[0];",
+            "  int negative4() {",
+            "    return 0;",
             "  }",
             "",
-            "  int negative5() {",
-            "    int[] arr = new int[0];",
-            "    return arr[hashCode()];",
+            "  String negative5() {",
+            "    return \"foo\" + \"bar\";",
             "  }",
             "",
-            "  int negative6() {",
-            "    return 1 * 2;",
```

Comment:
Previously the bug checker would see this as a trivial computation, but not the Refaster rule. The former was right.